### PR TITLE
A faster-built and smaller Docker image

### LINKED ISSUE #2908: Docker image extremely slow to build
To build the Scout image from Dockerfile is extremely slow (today it took 13 minutes from my computer). I think it's due to the fact that numpy requires a long time to be built. We could perhaps base the Scout image on an image that has already a pre-installed numpy?

The current Dockerfile has a multi-stage build and I wonder why.. Was there some package that was difficult to install in just one go?

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR should be reviewed after fixing #2967 

This PR:
- Reduces the time to build the Scout Dockerfile from 4,5 minutes to 70 seconds (fix #2908)
- Reduces the size of the Scout image from 764 Mb to 380 Mb 

**How to test**:
1. Remove all local scout-related containers and images
2. Clear the Docker cache: `docker builder prune`
2. from master branch, build scout image with the command: `docker build -t scout .`
3. Write down the time it takes and the final size of the the image
4. Switch to this branch
5. Repeat step 1
6. Make sure that the image works, by running `docker-compose up` and going to localhost:5000

**Expected outcome**:
- The image from this branch should compile faster and should be smaller in size than the one from master branch

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
-# Copy app
-COPY . /usr/src/app
-RUN pip install --no-cache-dir --upgrade pip &&           \
-    pip wheel --no-cache-dir --no-deps     \
-        --wheel-dir /usr/src/app/wheels    \
-		Cython gunicorn  &&                \
-    pip wheel --no-cache-dir               \
-        --wheel-dir /usr/src/app/wheels    \
-        --editable .[coverage]
+FROM northwestwitch/python3.8-cyvcf2-venv:1.0 AS python-builder
```
Comment (dnil) [inline Dockerfile:4]: It would not hurt with a fork to cg / cg docker hub! 😸

**Comment by northwestwitch** [comment]
@dnil done as suggested, we have a [this new repo](https://github.com/Clinical-Genomics/docker-base-images) now that contain base images like the one used to base this Dockerfile!

File: Dockerfile [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
-# Copy app
-COPY . /usr/src/app
-RUN pip install --no-cache-dir --upgrade pip &&           \
-    pip wheel --no-cache-dir --no-deps     \
-        --wheel-dir /usr/src/app/wheels    \
-		Cython gunicorn  &&                \
-    pip wheel --no-cache-dir               \
-        --wheel-dir /usr/src/app/wheels    \
-        --editable .[coverage]
+FROM clinicalgenomics/python3.8-cyvcf2-venv:1.0 AS python-builder
```
Comment (dnil) [inline Dockerfile:4]: 🥇

File: Dockerfile [hunk 3]
```diff
@@ -1,60 +1,50 @@
...
 LABEL about.tags="WGS,WES,Rare diseases,VCF,variants,SNP,Next generation sequencing"
 LABEL about.license="MIT License (MIT)"
 
-# Run app on non-root user
-RUN useradd -m worker && mkdir -p /home/worker/app
-WORKDIR /home/worker/app
+# Install base dependencies
+RUN apt-get update && \
+     apt-get -y upgrade && \
+     apt-get -y install -y --no-install-recommends libpango-1.0-0 libpangocairo-1.0-0 && \
```
Comment (dnil) [inline Dockerfile:28]: Much cleaner like this - nice with the builder base!

**Review by dnil — APPROVED** [review]
🏎️ 💨

File: Dockerfile (hunk 3 above)
Comment (northwestwitch) [reply Dockerfile:28]: Thanks for the review! 🙏🏻

**Pull request merged**